hadoop/src/main/java/org/apache/carbondata/hadoop/CarbonInputFormat.java

```diff
@@ -208,17 +215,14 @@ public static void setSegmentsToAccess(Configuration configuration, List<String>
         .set(CarbonInputFormat.INPUT_SEGMENT_NUMBERS, CarbonUtil.getSegmentString(validSegments));
   }
 
-  /**
-   * Set list of files to access
-   */
-  public static void setFilesToAccess(Configuration configuration, List<String> validFiles) {
```

Comment:
it seems this method is added by some out side contributor recently to set the files to read, so better keep it